src/data/uma-nickname-requirements.csv

```diff
@@ -105,11 +105,11 @@
 "長距離GⅠと短距離GⅠをそれぞれ勝利する","Win a GI race of short and long distance each once"
 "ファン数が500000人以上になる","Earn a total of ≥500000 fans"
 "GⅠを10勝する","Win a GI race 10 times"
-"全てのトレーニングレベルが3以上で育成を完了する","Complete the training with all infrastructure at level ≥3"
-"短距離、マイル、中距離、長距離、ダートの重賞を\nそれぞれ1勝ずつする","Win a dirt big race of every distance each once"
+"全てのトレーニングレベルが3以上で育成を完了する","Complete the training with all trainings at level ≥3"
+"短距離、マイル、中距離、長距離、ダートの重賞を\nそれぞれ1勝ずつする","Win one of each graded Short, Mile, Medium, Long, and Dirt race"
```

Comment:
This sounds a bit weird, maybe something like "Win a graded race from every category (Short, Mile, Medium, Long, Dirt)"?

Reply:
Might get too long, unsure about that.

Reply:
```suggestion
"短距離、マイル、中距離、長距離、ダートの重賞を\nそれぞれ1勝ずつする","Win a graded Short, Mile, Medium, Long, and Dirt race"
```

This should be clear enough right?

Reply:
That sounds good to me.